src/server/models/Local.js

```diff
@@ -0,0 +1,68 @@
+const https = require('https');
+
+// const Schema = mongoose.Schema;
+//         const locationSchema = new Schema({
+//             Lat: String,
+//             Lng: String
+//         });
+
+//         const Location = mongoose.model('Location', locationSchema);
+
+//         const NewLocation = Location({
+//             Lat: this.latitude,
+//             Lng: this.longitude,
+//             useMongoClient: true
+//           });
+
+module.exports = class Local {
+    constructor(nome) {
+        this.nome = nome;
+        //this.requestCoords(nome);
+    }
+    setLatitude(latitude) {
+        this.latitude = latitude
+    }
+    setLongitude(longitude) {
+        this.longitude = longitude;
+    }
+
+    requestCoords(nome) {
+        let key = `0be76b3fccbe4f768c99643d9f603e84`;
```

Comment:
Não deixem a chave da API no código! Essa é a regra número um quando se tem uma key para fazer requisição. Qualquer pessoa que não é do projeto pode pegar ela e usar sem autorização.

Reply:
To revolvendo isso para eles, é algo que tenho que colocar no Travis também